packages/frontend/src/components/NavBar/ProjectSwitcher.tsx

```diff
@@ -355,20 +369,27 @@ const ProjectSwitcher = () => {
                                 type: ProjectType.PREVIEW,
                             }),
                         )) ? (
-                        <>
+                        <Box
+                            pos="sticky"
+                            bottom={0}
+                            bg="gray.9"
+                            sx={(theme) => ({
+                                // fixes scroll overlap
+                                boxShadow: `0 4px ${theme.colors.dark[9]}`,
```

Comment:
Is this causing the small back box bellow it?

<img width="496" alt="Screenshot 2024-11-01 at 10 39 19" src="https://github.com/user-attachments/assets/3fcc7a82-7e10-4c1e-881a-bb535acaa5e9">

Reply:
nice catch. it should be gray.9 instead. I was barely visible on my monitor

Reply:
@ZeRego fixed!